backend/src/main/java/com/wootech/dropthecode/service/MemberService.java

```diff
@@ -56,6 +59,10 @@ public void deleteMember(LoginMember loginMember) {
         member.delete(DELETED_USER_EMAIL, DELETED_USER_NAME, DELETED_USER_IMAGE_URL);
         save(member);
 
+        if (member.hasRole(Role.TEACHER)) {
```

Comment:
```suggestion
        if (member.isTeacher()) {
```
이런 것도 괜찮을 것 같기도..? 개인 취향이라 반영하지 않으셔도 됩니당

Reply:
오 이것도 좋네요!

Reply:
바로 솔깃해서 위처럼 바꾸려고 했는데 hasRole(Role.Student)와 같이 사용하는 코드가 있네요...

그리고 Role 상태가 추가될때마다 isXXX가 추가될 거 같아서 일단은 유지하는 게 어떤가요?